strapi/strapi
# chore: upgrade better-sqlite3 package

### What does it do?

Use the latest better-sqlite3 package in all our templates, example projects, codemods, etc

### Why is it needed?

For some reason we were using 9.0.0 when they're already on 9.4.3. We intended to upgrade before releasing v5, so now is a good time.

### How to test it?

- new projects using sqlite should now set version 9.4.3
- codemod for sqlite3 -> better-sqlite3 should also use 9.4.3
- sqlite projects should work fine with that version

### Related issue(s)/PR(s)
